Make it possible to use Kedro after uninstalling Rich. (#3838)
kedro-org/kedro: RELEASE.md

# Upcoming Release 0.19.6

## Major features and improvements

> * It is now possible to uninstall the `rich` library from Kedro and have it maintain functionality.

Reviewer: ```suggestion
* It is now possible to use Kedro without having `rich` installed.
```
